rt/rt/OPCD$.MLC

```diff
@@ -1,13 +1,54 @@
+OPCD$DOS  TITLE 'Validate opcode table'
```

Comment:
Should "OPCD$DOS" be changed to "OPCD$"? If yes, same change on the CSECT and USING lines below.

Reply:
Correct, done.